Magento2/ruleset.xml

```diff
@@ -16,11 +16,6 @@
         <exclude-pattern>*\.phtml$</exclude-pattern>
         <exclude-pattern>*\.xml$</exclude-pattern>
     </rule>
-    <rule ref="Generic.PHP.DeprecatedFunctions">
```

Comment:
This only catches deprecations and, due to it using `get_defined_functions()`to get deprecated methods, its coverage is not as good as the solution from PHPCompatibility because it doesn't include extensions coming from functions or deprecations from different PHP versions than the one which is running the sniff.